src/Validation/Validation.php

```diff
@@ -301,41 +301,47 @@ public static function comparison($check1, $operator, $check2)
         switch ($operator) {
             case 'isgreater':
                 deprecationWarning(sprintf($message, $operator, 'COMPARE_GREATER'));
+                // @deprecated 3.6.0 Use Validation::COMPARE_GREATER instead.
```

Comment:
I would have switched the order of comment and deprecationWarning to call to not confuse this comment and the code below.

    // deprecated
    deprecationWarning()

Reply:
@dereuromark I think that fixer requires that the comment is the last line in a block

I get `There must be a comment when fall-through is intentional in a non-empty case body` when I change the order.

Reply:
Ah I see, ok.